easybuild/easyconfigs/m/MONAI/MONAI-1.4.0-foss-2023b-minimal.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonPackage'
+
+name = 'MONAI'
+version = '1.4.0'
+versionsuffix = '-minimal'
```

Comment:
@pavelToman Can you check if this is still useful, since there's also a PR for full MONAI now in #21937 ?